ark/phenotyping/som_utils.py

```diff
@@ -267,6 +264,11 @@ def train_som(fovs, channels, base_dir,
     # run the SOM training process
     process_args = ['Rscript', '/create_som_matrix.R', ','.join(fovs), ','.join(channels),
                     str(num_passes), subsetted_path, norm_vals_path, weights_path]
+
+    # add a random seed to function call if set
+    if seed is not None:
```

Comment:
I think it would be better to just set a default seed in the function definition

Reply:
My only concern is that having a default seed makes it less intuitive for the user to ask for a fully stochastic run if they wish (they'd have to know to pass in `None`). That being said, if we don't imagine many lab members will need this feature, better to remove it.

Reply:
Yeah, I think only Candace would want something like that for testing purposes